pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "isamples_metadata"
-version = "0.2.6"
+version = "0.2.7"
```

Comment:
Bump the version as that appears to be the only way to get Poetry to pick up the new dependency.